src/tools/curve_format_main.cpp

```diff
@@ -175,6 +179,25 @@ int AllocateFiles(AllocateStruct* allocatestruct) {
     return *allocatestruct->checkwrong == true ? 0 : -1;
 }
 
+// for clone chunk and snapshot chunk, we have bitmap to identify whether
+// corresponding block is written or not,
+// so, check whether meta page size is enough for store bitmap
+// and other metadata
+static bool CheckChunkAndMetaPageSize() {
+    if (!curve::common::is_aligned(FLAGS_fileSize, FLAGS_blockSize)) {
+        LOG(ERROR) << "io alignment or file size is wrong";
+        return false;
+    }
+
+    auto bitmapBytes = FLAGS_fileSize / FLAGS_blockSize / 8;
+    if (bitmapBytes > FLAGS_metaPageSize * 2) {
```

Comment:
If bitmapBytes > (FLAGS_metaPageSize-specified field), it should not work?
Here means if bitmapBytes = 4k, FLAGS_metaPageSize = 1k，that's ok？

Reply:
fixed, I added a hard code value `4096 - 512` to indicate minimum size usage for encoding and storing source location for clone chunk, so that, the meta page size should be great than or equal to `3584 + bitmap bytes`